src/Runner.Worker/Handlers/OutputManager.cs

```diff
@@ -139,12 +139,10 @@ public void OnDataReceived(object sender, ProcessDataReceivedEventArgs e)
 
                         // Convert to issue
                         var issue = ConvertToIssue(match);
-
                         if (issue != null)
                         {
                             // Log issue
-                            _executionContext.AddIssue(issue, stripped);
-
+                            _executionContext.AddIssue(issue);
```

Comment:
`ConvertToIssue` now leverages the original text stored within `match` so we no longer need to pass a log message override to `AddIssue`